bVNC/src/main/java/com/iiordanov/bVNC/dialogs/GetTextFragment.java

```diff
@@ -90,6 +91,7 @@ public void afterTextChanged(Editable arg0) {}
     private EditText textBox;
     private EditText textBox2;
     private EditText textBox3;
+    private Switch switch1;
```

Comment:
Why is this switch numbered? Is there a specific function that it fulfills or is its function generic? If there is a specific function, let's please name it more specifically!